==============================================================================
20160512 21:49:39.822 - INFO - + START SUITE: Local Artisan Interface Test [ ]
==============================================================================
20160512 21:49:39.830 - INFO - +- START SUITE: Local Artisan Interface Test.Server Api Test [ ]
==============================================================================
20160512 21:49:39.830 - INFO - +-- START TEST: get_crashTotal_current [ ]
------------------------------------------------------------------------------
20160512 21:49:39.831 - INFO - +--- START KW: common_keywords.get_CrashTotal_current [ ${app_id} ]
20160512 21:49:39.832 - INFO - +---- START KW: RequestsLibrary.Create Session [ event_query | ${query_ip} ]
20160512 21:49:39.832 - INFO - Creating Session using : alias=event_query, url=http://test-artisan.testbird.com:10086, headers={},                     cookies=None, auth=None, timeout=None, proxies=None, verify=False,                     debug=0
20160512 21:49:39.832 - DEBUG - Creating session: event_query
20160512 21:49:39.833 - DEBUG - Argument types are:
<type 'int'>
20160512 21:49:39.833 - INFO - +---- END KW: RequestsLibrary.Create Session (1)
20160512 21:49:39.833 - INFO - +---- START KW: RequestsLibrary.Get Request [ event_query | /issueCrashReporterTotal?app%3D${appkey} ]
20160512 21:49:39.842 - INFO - Starting new HTTP connection (1): test-artisan.testbird.com
20160512 21:49:39.927 - INFO - Get Request using : alias=event_query, uri=/issueCrashReporterTotal?app%3D0o3Nwynm, headers=None
20160512 21:49:39.927 - INFO - ${response} = <Response [404]>
20160512 21:49:39.927 - INFO - +---- END KW: RequestsLibrary.Get Request (94)
20160512 21:49:39.928 - INFO - +---- START KW: RequestsLibrary.To Json [ ${response.content} ]
20160512 21:49:39.930 - FAIL - ValueError: No JSON object could be decoded
20160512 21:49:39.930 - DEBUG - Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/RequestsLibrary/RequestsKeywords.py", line 307, in to_json
    json_ = json.loads(content)
  File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 339, in loads
    return _default_decoder.decode(s)
  File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/decoder.py", line 364, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/decoder.py", line 382, in raw_decode
    raise ValueError("No JSON object could be decoded")
20160512 21:49:39.931 - INFO - +---- END KW: RequestsLibrary.To Json (2)
20160512 21:49:39.931 - INFO - +--- END KW: common_keywords.get_CrashTotal_current (100)
------------------------------------------------------------------------------
20160512 21:49:39.932 - INFO - +-- END TEST: get_crashTotal_current (102)
------------------------------------------------------------------------------
20160512 21:49:39.934 - INFO - +- END SUITE: Local Artisan Interface Test.Server Api Test (110)
==============================================================================
20160512 21:49:39.936 - INFO - + END SUITE: Local Artisan Interface Test (235)
==============================================================================
